README.md

```diff
@@ -1,24 +1,38 @@
 # JavaScript Comparisons
 
-## Overview
-In addition to performing arithmetic and assigning value to variables, JavaScript has additional operators for comparing values. The value returned by a comparison is **always** `true` or `false`.
-
-## Objectives
-1. Compare values with `===` and `!==` to test for equality.
-2. Understand the difference between loose and strict equality and why `==` and `!=` should never be used.
-3. Test how values relate to one another with `>`, `>=`, `<`, and `<=`.
-4. Explain why it's best to only compare numbers with the relational operators.
-
-## Equality operators
-
-### `==`
+## Problem Statement 
+Now we know about performing basic mathematical functions, and assigning values
+to variables, but how would we check to see if a value is what we're expecting?
+In addition to performing arithmetic and assigning value to variables,
+JavaScript has additional operators for comparing values. The value returned by
+a comparison is **always** `true` or `false`.
+
+## Objectives 
+1. Identify equality operators
+2. Compare numbers with the relational operators
+
+## Identify equality operators 
+There are four equality operators built into Javascript:
+- **loose equality operator** (`==`)
+- **strict equality operator** (`===`)
+- **loose inequality operator** (`!=`)
+- **strict inequality operator** (`!==`)
+
+When writing Javascript, you only ever use the **strict** operators, as the
```

Comment:
`When writing Javascript, you only ever use the **strict** operators...`
Is there a case where this is ever not true? Should we instead strongly suggest/urge students to vs. saying that this is _always_ the case?